drivers/overlay/ov_network.go

```diff
@@ -173,9 +256,11 @@ func (n *network) initSandbox() error {
 	})
 
 	go n.watchMiss(nlSock)
-	return n.initVxlan()
+	return nil
+	//return n.initVxlan()
```

Comment:
@mrjana I left this one intentionally because it wasn't clear to me why this function initVxlan was added. There was only one caller. I have now moved what initVxlan was doing into joinSubnetSandbox(). So if these changes look ok I will remove the commented lines.